sklearn/compose/_column_transformer.py

```diff
@@ -55,7 +55,8 @@ class ColumnTransformer(_BaseComposition, TransformerMixin):
         name : string
             Like in Pipeline and FeatureUnion, this allows the transformer and
             its parameters to be set using ``set_params`` and searched in grid
-            search.
+            search. The name 'remainder_transformer' can not be used if the
```

Comment:
I don't think this is necessary. The name `remainder` can't be used in any case.